include/coreinit/memexpheap.h

```diff
@@ -131,6 +131,9 @@ MEMVisitAllocatedForExpHeap(MEMHeapHandle heap,
                             MEMExpHeapBlockVisitor callback,
                             void *context);
 
+bool
+MEMCheckExpHeap(MEMHeapHandle handle, bool logErrors);
```

Comment:
The `logErrors` parameter seems more like flag bits to me, since the function only checks for the lowest bit to log errors.

Reply:
Fixed